include/rocksdb/utilities/env_mirror.h

```diff
@@ -33,9 +33,21 @@ class WritableFileMirror;
 
 class EnvMirror : public EnvWrapper {
   Env* a_, *b_;
+  bool free_a_, free_b_;
```

Comment:
Where are we going to use these options ? 
I don't see we are calling them anywhere